rclc/include/rclc/node.h

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) 2019 - for information on the respective copyright owner
+// see the NOTICE file and/or the repository https://github.com/ros2/rclc.
+// Copyright 2014 Open Source Robotics Foundation, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef RCLC__NODE_H_
+#define RCLC__NODE_H_
+
+#if __cplusplus
+extern "C"
+{
+#endif
+#include <rcl/node.h>
+#include <rclc/types.h>
+#include <rclc/init.h>
```

Comment:
```suggestion

#include <rclc/init.h>
```

Reply:
fixed in 54f0bbc54a33cc601764ab1d53fe3119d9e61c8b